### DISCUSSION TITLE
plays/alaminrifat/url-shortner

### DISCUSSION THREAD

**Discussion by giscus** [reactplay/react-play#1491] [Play Feedback]
# plays/alaminrifat/url-shortner

URL shortener will takes a long URL and generates a shorter, more manageable version of it.

https://reactplay.io/plays/alaminrifat/url-shortner

**Comment by karimkkanji** [comment]
This is a really good easy to use project. Just something to note as an improvement for the UX: Let's add a loader to the "Shorten This Link" button and disable it when one clicks on it until we get a response from the TinyURL API, so that I know that the button actually works. I was surprised I clicked the button three times until I saw the three toasts, meaning, I hit the endpoints three times as I was waiting for visual response. Hope to see how you implement this!
